gosushi_client/src/components/MenuSelection.jsx

```diff
@@ -73,7 +73,7 @@ const MenuSelection = ({ handleMenu, menu, numPlayers }) => {
   };
 
   const selectDessert = item => {
```

Comment:
I'm not a huge fan of the selectRoll, selectAppetizers, selectSpecial and selectDessert functions. I think we could get rid of the some code duplication by replacing those functions with something like:

For specials and apps:
`const selectMultiItemCourse = (item, course, maxItems) => {
    if (!validMenuOption(item)){
      return;
    }

    let menucopy = _.cloneDeep(menu);
    let newCourse = menu[course].slice();
    if (menu[course].includes(item)) {
      _.remove(newCourse, i => i === item);
      menucopy[course] = newCourse;
    } else if (menu[course].length < maxItems) {
      newCourse.push(item);
      menucopy[course] = newCourse;
    }
    handleMenu(menucopy);
  };`

For dessert and roll:
  `const selectSingleItemCourse = (item, course) => {
    let menucopy = _.cloneDeep(menu);
    menucopy[course] = item;
    handleMenu(menucopy);
  }`

and then replacing their calls from `selectSpecial(item)` to basically `selectMultiItemCourse(item, 'specials', 3)`.